Source/Core/DolphinQt2/Debugger/CodeViewWidget.cpp

```diff
@@ -166,9 +165,16 @@ void CodeViewWidget::Update()
     }
   }
 
-  g_symbolDB.FillInCallers();
+  for (int i = 1; i < columnCount(); i++)
```

Comment:
`++i`

Reply:
Really?

Reply:
Yep, going by the last bullet [here](https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md#general-1).

Reply:
I'm not going to comment on this further for obvious reasons.

Reply:
I'll likely open a PR to clarify that styling bullet point. The intention behind prefixed `++` is when a non-trivial type has an overloaded `operator++(int)` and an overloaded `operator++()`, it's usually preferable to use the prefixed overload if a temporary isn't necessary. e.g. when dealing with std lib iterators, etc.

When dealing with primitive built-in types, it's just a matter of preference, considering a compiler should be able to trivially optimize this case regardless of whether or not it's prefix-form or postfix-form